packages/web-components/src/global/color-mode.css

```diff
@@ -361,6 +378,20 @@ ic-theme,
     --ic-chip-action-hover: var(--ic-action-light-bg-hover);
     --ic-chip-action-pressed: var(--ic-action-light-bg-pressed);
 
+    /* Card vertical */
+    --ic-card-vertical-card-text: var(--ic-color-text-primary);
+    --ic-card-vertical-disabled-text: var(--ic-color-text-tertiary);
```

Comment:
The disabled text doesn't quite match the designs - I think this should be `var(--ic-color-text-secondary-light)`?

And line 640